mlflow/server/js/src/components/artifact-view-components/ShowArtifactImageView.js

```diff
@@ -35,8 +35,19 @@ class ShowArtifactImageView extends Component {
     return getSrc(path, runUuid);
   };
 
+  isGif = () => {
+    return this.props.path.endsWith('.gif');
+  };
+
   fetchImage = () => {
     this.setState({ loading: true });
+
+    // gif
+    if (this.isGif()) {
+      return;
```

Comment:
Could we add the explanation in the PR description here as well?

Reply:
Actually, this is the part where I want to hear your thoughts. This is related to `Plotly` behavior. `Plotly` doesn't seem to provide a callback function that's called when image loading is done. That's why I load an image here, convert it to data URI and let `Plotly` just render it to manage the loading state. But, for gif, we don't have to do that because `img` tag can take care of that with `onLoad`.

Reply:
Oh, what I mean is we should add a comment here, similar to your explanation in the PR description. The approach LGTM.

Reply:
@Zangr Oh, I misunderstood what you meant. I'll add comments.